src/locales/en/fields.yml

```diff
@@ -3,21 +3,31 @@ drivingLicenceNumber:
   hint: "This is the long number in section 5 on your licence for example HARRI559146MJ931 "
   validation:
     default: "Enter the number as it appears on your driving licence"
-    exactlength: "Your driving licence number should be 8-16 digits long"
-    numeric: "Your driving licence number should not include letters or symbols"
+    exactLength: "Your licence number should be 16 characters long"
+    numeric: "Your licence number should not include any symbols or spaces"
+    limit: "Driving licence numbers do not start with 16 – enter the number as it appears on your driving licence"
+dvaLicenceNumber:
+  label: Driving Licence number
+  hint: "This is the 8 digit number in the top right hand corner of the photo page in your driving licence"
+  validation:
+    default: "Enter the number as it appears on your driving licence"
+    exactLength: "Your licence number should be 8 characters long"
+    numeric: "Your licence number should not include any symbols or spaces"
     limit: "Driving licence numbers do not start with 16 – enter the number as it appears on your driving licence"
 issueNumber:
   label: Issue Number
   hint: "This is the 2 digit number after the space in section 5 of your licence."
   validation:
-    default: "Enter the number as it appears on your driving licence"
+    default: "Enter the issue number as it appears on your driving licence"
+    exactLength: "Your issue number should be 2 numbers long"
```

Comment:
Do we not need a validation to trigger this

Reply:
Added.